health/health.go

```diff
@@ -170,6 +170,20 @@ func Register(name string, check Checker) {
 	registeredChecks[name] = check
 }
 
+// Unregister removes the named checker.
+func Unregister(name string) {
+	mutex.Lock()
+	defer mutex.Unlock()
+	delete(registeredChecks, name)
```

Comment:
Is Unregister expected to call in runtime? I'm not sure if it's necessary, but Checks could leave various footprints, (for example, goroutines). May be we should add Close method to Checker interface?